libr/socket/run.c

```diff
@@ -166,9 +167,9 @@ static char *getstr(const char *src, R_NULLABLE size_t *out_len) {
 	size_t len = 0;
 	char *ret = NULL;
 
-	switch (*src) {
+	switch (*(src++)) {
```

Comment:
```suggestion
	src++;
	switch (*src) {
```

Reply:
imho its more readable in two separate lines

Reply:
This is different behavior though, I need to read first and then increment the pointer